lazy_tensor_core/lazy_tensor_core/csrc/init_python_bindings.cpp

```diff
@@ -683,6 +683,63 @@ void InitLtcModuleBindings(py::module m) {
   m.def("_ltc_set_noop_execution_mode", [](bool enable_noop) {
     torch::lazy::LazyGraphExecutor::Get()->SetNoOpExecutionMode(enable_noop);
   });
+  m.def("_jit_function_call", [](py::args args, py::kwargs kwargs) {
+    HANDLE_TH_ERRORS
+
+    auto strongPtr = py::cast<torch::jit::StrongFunctionPtr>(args[0]);
+    torch::jit::Function& callee = *strongPtr.function_;
+
+    std::cerr << "function pointer = " << &callee << std::endl;
+
+    std::vector<c10::IValue> stack;
+    size_t tensor_args = 0;
+
+    std::cerr << "constructing arguments\n";
+    torch::lazy::BackendDevice device;
+    std::vector<torch::lazy::Value> tvals;
+
+    std::vector<c10::IValue> ivals;
+
+    // replace lazy tensors with meta tensors
+    // so we can do shape inference
+    for (auto i: c10::irange(args.size() - 1)) {
+      if (THPVariable_Check(args[i + 1].ptr())) {
+        auto& t = THPVariable_Unpack(args[i + 1].ptr());
+        auto lt = torch::lazy::GetLtcTensor(t);
+        device = lt.GetDevice();
+        tvals.push_back(lt.GetIrValue());
+        auto mt = at::empty_strided(t.sizes(), t.strides(), at::TensorOptions(at::kMeta).dtype(t.scalar_type()));
+        stack.push_back(mt);
+      } else {
+        ivals.push_back(torch::jit::toTypeInferredIValue(args[i+1]));
```

Comment:
just to be clear, this case is for scalars?

Reply:
yes